pkg/runners/builtin_runner/bitwise.go

```diff
@@ -0,0 +1,95 @@
+package builtinrunner
+
+import (
+	"errors"
+
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+)
+
+const CELLS_PER_INSTANCE = 5
+const TOTAL_N_BITS = 251
+const INPUT_CELLS_PER_INSTANCE = 2
```

Comment:
```suggestion
const BITWISE_CELLS_PER_INSTANCE = 5
const BITWISE_TOTAL_N_BITS = 251
const BIWISE_INPUT_CELLS_PER_INSTANCE = 2
```
As all the builtins are in the same package, constant's names can clash